src/services/services.ts

```diff
@@ -5762,6 +5773,62 @@ namespace ts {
             return forEach(diagnostics, diagnostic => diagnostic.category === DiagnosticCategory.Error);
         }
 
+        function getDependencies(fileName: string): DependencyInfo {
+            synchronizeHostData();
+            
+            let sourceFile = getValidSourceFile(fileName);
+            if (!sourceFile) {
+                return undefined;
+            }
+            
+            let resolver = program.getDiagnosticsProducingTypeChecker().getEmitResolver(sourceFile);
+            
+            let runtime: string[] = [];  
+            let compileTime: string[] = [];  

```

Comment:
initially i thought that compileTime is a subset of runtime, but then looking at the code, that did not turn out to be the case. i think we need better names here. maybe "compileTimeOnly", or "non-js dependencies.."